python/doc/theory/numerical_methods/rosenblatt_transformation.rst

```diff
@@ -3,51 +3,54 @@
 Rosenblatt Transformation
 -------------------------
 
-| The Rosenblatt transformation is an isoprobabilistic transformation
-  (refer to ) which is used under the following context :
-  :math:`\vect{X}` is the input random vector, :math:`F_i` the
-  cumulative density functions of its components and :math:`C` its
-  copula, without no condition on its type.
-| Let us denote by :math:`\vect{d}` a deterministic vector,
-  :math:`g(\vect{X}\,,\,\vect{d})` the limit state function of the
-  model,
-  :math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-  the event considered here and g(,) = 0 its boundary.
-| One way to evaluate the probability content of the event
-  :math:`\cD_f`:
-
-  .. math::
+The Rosenblatt transformation is an :ref:`isoprobabilistic transformation <isoprobabilistic_transformation>` which is used under
+the following context: the input random vector is :math:`\vect{X}` with marginal cumulative
+density functions  :math:`F_i` and copula :math:`C`. Nothing special is assumed about the
+copula.
+
+Let :math:`\vect{d}` be a  deterministic vector, let :math:`g(\vect{X}\,,\,\vect{d})` be the
+limit state function of the model and let :math:`\cD_f = \{\vect{X} \in \Rset^n \,/ \,
+g(\vect{X}\,,\,\vect{d}) \le 0\}` be an event whose probability :math:`P_f` is defined as:
+
+.. math::
     :label: PfX2
 
       P_f = \Prob{g(\vect{X}\,,\,\vect{d})\leq 0}=   \int_{\cD_f}  \pdf\, d\vect{x}
 
-  is to use the Rosenblatt transformation :math:`T` which is a
-  diffeomorphism from :math:`\supp{\vect{X}}` into the standard space
-  :math:`\Rset^n`, where distributions are normal, with zero mean, unit
-  variance and unit correlation matrix (which is equivalent in that
-  normal case to independent components).
+One way to evaluate the probability :math:`P_f` is to use the Rosenblatt transformation :math:`T` which is a
+diffeomorphism from the support of the distribution :math:`f_{\vect{X}}` into the standard
+space :math:`\Rset^n`, where distributions are normal, with zero mean, unit
```

Comment:
To me the idea of a space "where distributions are normal" makes no sense. That is why I suggested viewing the standard normal distribution on that space as the push-forward of f_X by the transformation.

Reply:
Link to the suggestion: https://github.com/openturns/openturns/pull/2262#discussion_r1170036023

Reply:
The Rosenblatt transformation maps to a space where the variables have a normal distribution with independent components. In the text, I can precise the 'Rosenblatt- standard space' rather than the 'standard space'.